themes/custom/droopler_theme/scss/components/_d_search_blocks.scss

```diff
@@ -0,0 +1,19 @@
+// Colour variables used in this file.
+$d-search-page-link-color: $color-iphis;
+$d-search-page-link-color-hover: $link-hover-color;
+
+.block-search-page-link {
+  .search-page-link {
+    margin: 5px 10px 0 10px;
+    display: block;
+    width: 24px;
+    height: 24px;
```

Comment:
i don't hate it :point_up: it's rather a question i always ask myself pushing some code here.. "What would @grzegorz-pietrzak-droptica do?" 
usually every value in scss here is put into variables first..like the colors etc
you know what i mean :point_down: 
![image](https://user-images.githubusercontent.com/37533838/68795183-b7dbb100-0650-11ea-887b-1da8712a1815.png)